roundtripper_test.go

```diff
@@ -107,12 +107,12 @@ func TestRoundTripper_TransportFailureErrorHandling(t *testing.T) {
 
 	expectedError := &url.Error{
 		Op:  "Get",
-		URL: "http://this-url-does-not-exist-ed2fb.com/",
+		URL: "http://asdfsa.com/",
```

Comment:
I don't really understand why, but this was returning a 200 for most URLs, but for whatever reason did not when I typed some gibberish. It seemed potentially related to the ISP since when I went to `this-url-...` in my browser it redirected to some sort of AT&T search page for possible other pages.

Not sure if there is a better way to test this.